doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,187 @@
+
+.. code-block:: text
+
+    subsection model parameters
+      set contact detection method 		   	 = dynamic
+      set dynamic contact search size coefficient	 = 0.8
+      set neighborhood threshold			 = 1.3
+      set load balance method				 = dynamic
+      set load balance threshold			 = 0.5
+      set dynamic load balance check frequency		 = 100000
+      set particle particle contact force method         = hertz_mindlin_limit_overlap
+      set particle wall contact force method             = nonlinear
+      set integration method				 = velocity_verlet
+    end
+
+Boundary Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The rotation of the cylinder is applied using a rotational boundary condition with a value of 8 rad/s over the x axis. Based on Deal.II boundary colouring, the hull of the cylinder (rotating drum) has an id = 0.
+
+.. code-block:: text
+
+    subsection DEM boundary conditions
+      set number of boundary conditions         = 1
+        subsection boundary condition 0
+            set boundary id					= 0
+            set type              				= rotational
+            set rotational speed				= 8
+            set rotational vector x				= 1
+            set rotational vector y				= 0
+            set rotational vector z				= 0
+        end
+    end
+
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The end time of the simulation is 10 seconds in real time. We set the time equal to 12 seconds as the simulation is restarted after the packing dem simulation which was run for 2 seconds in real time.
+
+.. code-block:: text
+
+    subsection simulation control
+      set time step                 		 = 5e-6
+      set time end       			 = 12
+      set log frequency				 = 2000
+      set output frequency            		 = 2000
+      set output path                  	 	 = ./output_dem/
+    end
+
+Running the simulation
+-----------------------
+
+The simulation is launched on two steps, the first step packs the particle in the cylinder. The second step rotates the drum and simulates the movement of the particles. The simulation could have been performed in a single step, however; doing it this way allows us to use the restart files of the packing to run the simulation using DEM for a dry simulation and a CFD-DEM for a wet simulation without having to pack the particles twice.
+
+For packing:
+
+.. code-block:: text
+
+   mpirun -np 6 dem_3d packing_rotating_drum.prm
+   mpirun -np 6 dem_3d small_rotating_drum_dem.prm
+
+
+.. warning::
+	This example needs a simulation time of approximately 6 hours on 6 cores.s.
+
+
+Results
+---------
+
+Velocity profile of particles during the simulation:
+
+.. image:: image/rotating_drum.png
```

Comment:
There are also other things you could analyse. 
- Make a movie of the results instead of a static image. that would be fun
- Make a few images of the processor distribution in the domain for a few times, that would be cool to show
:)